CHANGELOG.md

```diff
@@ -1,8 +1,23 @@
 # Changes in Fbx Exporter
 
-## [Unreleased]
+## [5.0.0-pre.1]
```

Comment:
@vkovec the entry is missing the date, so package validation is failing because of that.
Thanks

Reply:
I fixed it ✋

Reply:
thank you!